contrib/win32/win32compat/tnnet.c

```diff
@@ -82,14 +83,12 @@ processBuffer(HANDLE handle, char *buf, size_t len, unsigned char **respbuf, siz
 			gbVTAppMode = false;
 
 		/* Console has the capability to parse so pass the raw buffer to console directly */
-		ConRestoreViewRect(); /* Restore the visible window, otherwise WriteConsoleW() gets messy */
```

Comment:
Why is this removed?

Reply:
It's not required now.
I added this earlier to mitigate the conpty bug where if we scroll up and start typing then it will print the characters in the scroll up window instead of the last line..  Now conpty has fixed it so this code is not required any more for RS5.

Also this code (if exists) will block any one from maximizing their window as it will restore the window back. Again this is due to the conpty bug which is now fixed.